src/components/htmlMediaHelper.js

```diff
@@ -231,6 +252,20 @@ import Events from '../utils/events.ts';
         }
     }
 
+    export function destroyShakaPlayer(instance) {
+        const player = instance._shakaPlayer;
+        if (player) {
+            instance._shakaPlayer = null;
+            try {
+                return player.destroy();
+            } catch (err) {
+                console.error(err);
+            }
```

Comment:
tbh, I don't know what it returns if an exception occurs in `destroy`. This would be better, imo,
```suggestion
            return player.destroy().catch((err) => {
                console.error(err);
            });
```